src/js_native_api_v8.h

```diff
@@ -315,6 +315,7 @@ inline napi_value JsValueFromV8LocalValue(v8::Local<v8::Value> local) {
 
 inline v8::Local<v8::Value> V8LocalValueFromJsValue(napi_value v) {
   v8::Local<v8::Value> local;
+  static_assert(sizeof(v) <= sizeof(local), "size of napi_value is larger than V8 Local Value");
```

Comment:
They are expected to be the same size.

```suggestion
  static_assert(sizeof(v) == sizeof(local), "size of napi_value is not equal to v8::Local<Value>");
```